index.html

```diff
@@ -49,43 +50,48 @@
             </div>
         </div>
         <div class="site-hero">
-        <div class="fling-minislide">
-            <img src="images/zdj1.jpg" alt="Slide 4" />
-            <img src="images/zdj3.jpg" alt="Slide 3" />
+            <div class="fling-minislide">
+                <img src="images/zdj1.jpg" alt="Slide 4" />
+                <img src="images/zdj2.jpg" alt="Slide 3" />
 
-            <img src="images/zdj2.jpg" alt="Slide 2" />
+                <img src="images/zdj3.jpg" alt="Slide 2" />
 
-            <img src="images/zdj1.jpg" alt="Slide 1" />
-        </div>
+                <img src="images/zdj4.jpg" alt="Slide 1" />
+            </div>
 
         </div>
 
-<div class="mainContainer">
-    <div class="description">
-        <h1 class="content__header"> Co robimy z pasją?</h1><p class="paw">.</p>
-        <p class="content__description-title">Ratując jednego psa nie zmienisz świata, ale cały świat zmieni się dla tego jednego psa!</p>
-        
-        <p class="stars"></p>
-        
-        
-    </div>
-    
-    <div class="heroImage"></div>
-    <main class="content">
-        
-        <p class="content__description"> Fundacja 4 Code for Animals została założona w celu niesienia pomocy bezdomnym, skrzywdzonym zwierzętom.Początkowo Fundacja działała w niewielkich pomieszczeniach. Czworonogi otrzymywały niezbędną pomoc weterynaryjną. Wszystkie znajdowały nowe, kochające domy. Zwierząt przybywało... </p>
-       
-        <p class="content__description">Dlatego każdego dnia ratujemy zwierzaki porzucone i zaniedbane. Umieszczamy w tzw. domach tymczasowych u naszych przyjaciół, leczymy, szczepimy, odkarmiamy i szukamy psiakom nowych domów... W normalnych, nieschroniskowych warunkach taki pies ma szansę otworzyć się- wówczas łatwiej znaleźć dla niego odpowiednich i kochających opiekunów. 
-        </p>
-            <p class="content__description">Nie ma nic piękniejszego niż obserwowanie jak pies rozkwita pod czułą opieką swojego opiekuna... .</p>
-    
+        <div class="mainContainer">
+            <div class="description">
+                <h1 class="content__header"> Co robimy z pasją?</h1>
+                <p class="paw">.</p>
```

Comment:
po co ta kropka? przecież widać ją na stronie ;p i wygląda to tak jakby się miało brudny ekran :D sam się na to nadziałem :P